### ISSUE TITLE
kore-repl log options not working

### ISSUE THREAD

**Issue by ana-pantilie** [kframework/kore#1843]
The logging command-line options should also work with `kore-repl`. We should investigate which options don't work and fix them.

**Comment by andreiburdusa** [comment]
What I've noticed so far by playing with the integration test `kprove/sum-spec.k`:

If I run kore-repl with the flags `--log-level debug --disable-log-timestamps --debug-equation _+Int_ ` it starts by showing messages like
```
kore-repl: Debug (DebugSolverSend):
    (DebugEvaluateCondition) while evaluating predicate:
    (check-sat )
kore-repl: Debug (DebugSolverRecv):
    (DebugEvaluateCondition) while evaluating predicate:
    sat
```
so the first two flags seems to work.

However, the last one doesn't make any difference until the end of the proof. But if I set the [`state :: ReplState`](https://github.com/kframework/kore/blob/master/kore/src/Kore/Repl.hs#L164) -> `koreLogOptions` ->`debugEquationOptions` to contain the identifier `_+Int_`, the first step of the proof also shows messages like
```
kore-repl: [2020-06-11 14:08:06.287646672] Debug (DebugAttemptEquation):
    (InfoReachability) While checking implication of the proof goal:
    equation is not applicable:
    could not apply match result:
        variable VarI did not match a concrete term
```
Note the timestamps are no longer disabled.

I'm thinking about setting the `state :: ReplState` to have the `KoreLogOptions` provided by the user instead of the default values.
